zerobuf/render/lookOut.fbs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Grigori.Chevtchenko@epfl.ch
+//
+namespace zerobuf.render;
+
+// This event is named "LookOut" instead of "Camera" because
+// it may be used in immersive environments. The orientation of 
+// a display surface or the head position of the observer should 
+// not be part of this event, i.e., if the user is on a magic 
+// carpet, you will synchronize the matrices describing the 
+// carpets but not the user's local movements on the carpet.
```

Comment:
More scientifically can be said that, this transforms from world space to carpet space ( Here worlds assumed to be same space )